src/pokemon.c

```diff
@@ -6669,21 +6669,20 @@ u16 GetEvolutionTargetSpecies(struct Pokemon *mon, u8 mode, u16 evolutionItem, s
                     u8 nature = GetNature(mon);
                     switch (nature)
                     {
-                        case NATURE_HARDY:
-                        case NATURE_BRAVE:
-                        case NATURE_ADAMANT:
-                        case NATURE_NAUGHTY:
-                        case NATURE_DOCILE:
-                        case NATURE_IMPISH:
-                        case NATURE_LAX:
-                        case NATURE_HASTY:
-                        case NATURE_JOLLY:
-                        case NATURE_NAIVE:
-                        case NATURE_RASH:
-                        case NATURE_SASSY:
-                        case NATURE_QUIRKY:
-                            targetSpecies = gEvolutionTable[species][i].targetSpecies;
-                            break;
+                    case NATURE_HARDY:
+                    case NATURE_BRAVE:
+                    case NATURE_ADAMANT:
+                    case NATURE_NAUGHTY:
+                    case NATURE_DOCILE:
+                    case NATURE_IMPISH:
+                    case NATURE_LAX:
+                    case NATURE_HASTY:
+                    case NATURE_JOLLY:
+                    case NATURE_NAIVE:
+                    case NATURE_RASH:
+                    case NATURE_SASSY:
+                    case NATURE_QUIRKY:
+                        targetSpecies = gEvolutionTable[species][i].targetSpecies;
```

Comment:
> missing a `break` here and down below

Is there? :eyes:

Doesn't the code leave the `switch` if the nature of the 'mon scanned doesn't match any of the cases inside the `switch`, and then it exists the `if (gEvolutionTable[species][i].param <= level)` to meet the `break` that is already there in both situations, the `case EVO_LEVEL_NATURE_AMPED`'s and the `case EVO_LEVEL_NATURE_LOW_KEY`'s? :eyes:

![sublime_text_20220701_1551688](https://user-images.githubusercontent.com/4485172/176953939-7eabff60-48ae-46ae-a0bf-46ab8cfecb68.png)

Reply:
Inside the inner switch statement, under the target species field. It's not strictly necessary since there are no cases below but it's good programming practice

Reply:
Right, I know that it'd go inside the switch statemet as to `break;` out of the cases, but I didn't see it necessaery because again, I thought the code would fall down into the `break;` outside of the `if` statement that the `switch` statements are located in.

Thanks for the insight though, will push in a moment :eyes: